integrationtests/src/test/java/org/axonframework/integrationtests/domain/ComplexAggregateStructureTest.java

```diff
@@ -148,7 +159,7 @@ public Paragraph(int paragraphId) {
 
```

Comment:
It would be wise to put some interceptors here, register their invocation in some static variable, so that their invocations can be tracked and validated.

Reply:
I will add few more tests to `CommandHandlerInterceptorTest`. `ComplexAggregateStructureTest` does not test full functionality of interceptors since `AnnotatedAggregate#handle` is invoked for constructors also, which is not the case in real life scenario.